tools/reviewer/web/src/app/dashboard/index.ts

```diff
@@ -2,33 +2,31 @@ export * from './reviews-panel/';
 
 import {
   AddCommentButtonsComponent,
-  BugComponent,
   ChangesHighlightingComponent,
   CodeBlockComponent,
   CommentsComponent,
   DiffComponent,
   DiffService,
+  EditablePropertyComponent,
   LineNumbersComponent,
   PersonListComponent,
   ReviewComponent,
-  ReviewsPanelComponent,
+  ReviewsPanelComponent
```

Comment:
I noticed that you don't like the extra comma in lists.
But think about it.
It's not an error and doesn't cause any issues. But this makes it easier to work with lists. Each line has the same structure. When you want to duplicate the last line (Ctrl + Shift + D in sublime) to add new one you don't need to add a comma to the previous line. It's convenient.
Also, it's pretty common practice.
Look, angular developers use it:
https://github.com/alexeagle/angular-bazel-example/blob/master/src/app.module.ts#L11

Reply:
Hey Vadim, I love discussing grammar :). I'm of the opinion that code should always follow grammar like it's speaking a language. The list should be alphabetized so that means most of the times the new entry will be some where in the middle of the list. If you want to add an entry at the end of the list, I always (Ctrl + Shift +D) the second last line in the list. It has the same effect of convenience. 

If you put a comma at the end of a json file it'll start giving error so it's a nice practice to always keep it nice and clean. A nice practice of dotting your i's crossing your t's and semicoloning your JavaScript statements :D